job-portal/routes/auth.js

```diff
@@ -21,7 +35,11 @@ router.post('/register', function (req, res) {
     var newUser = new User({
       email: req.body.username,
       password: req.body.password
+      
     })
+    if (passValidation(newUser.password) == false){
```

Comment:
So there's this thing in js called triple equal `===`. So it is almost the same as `==` but it also checks the **type**. Hence, change this line to `if (passValidation(newUser.password) === false){`